src/Surfnet/AzureMfa/Infrastructure/Form/EmailAddressModel.php

```diff
@@ -26,6 +26,7 @@ final class EmailAddressModel
     /**
      * @Assert\NotBlank
      * @Assert\Email
+     * @Surfnet\AzureMfa\Infrastructure\Validator\Constraint\EmailDomainInConfiguration
      */
     private $emailAddress = '';
```

Comment:
Please rename the class to *Dto or *Command instead *Model because this isn't a real (domain) model.

Reply:
Renamed to *Dto